tests/coreos-install/install_test.go

```diff
@@ -53,6 +53,9 @@ func TestCoreosInstall(t *testing.T) {
 		LocalAddress:   addr,
 	}
 
+	networkUnit := util.CreateNetworkUnits(t)
+	defer os.RemoveAll(networkUnit)
```

Comment:
As a matter of taste, let's not call `RemoveAll` on an empty string if returned by `CreateNetworkUnits`.